waspc/src/Wasp/AppSpec/Valid.hs

```diff
@@ -193,3 +194,12 @@ getDbSystem spec = AS.Db.system =<< AS.App.db (snd $ getApp spec)
 -- | This function assumes that @AppSpec@ it operates on was validated beforehand (with @validateAppSpec@ function).
 isPostgresUsed :: AppSpec -> Bool
 isPostgresUsed = (Just AS.Db.PostgreSQL ==) . getDbSystem
+
+-- | This function assumes that @AppSpec@ it operates on was validated beforehand (with @validateAppSpec@ function).
+-- If there is no user entity, it returns Nothing.
+doesUserEntityContainField :: AppSpec -> String -> Maybe Bool
+doesUserEntityContainField spec fieldName = do
+  auth <- App.auth (snd $ getApp spec)
+  let (_userEntityName, userEntity) = AS.resolveRef spec (Auth.userEntity auth)
```

Comment:
You could go for `snd` like above :)

Reply:
I didn't have a strong pref so I just updated with suggestion :) 👍🏻